src/lib/config/arbitrum/pools.ts

```diff
@@ -142,6 +142,15 @@ const pools: Pools = {
     '0xd3d5d45f4edf82ba0dfaf061d230766032a10e07000200000000000000000413', // stg/ bb-a-usd
     '0x9fb7d6dcac7b6aa20108bad226c35b85a9e31b63000200000000000000000412', // wsteth-bb-a-weth / bb-a-usd
   ],
+  Deprecated: {
+    '0x178e029173417b1f9c8bc16dcec6f697bc323746000200000000000000000158': {
+      newPool:
+        '0x9fb7d6dcac7b6aa20108bad226c35b85a9e31b63000200000000000000000412 ',
+      description: 'deprecatedPool.migrateCardDescription',
```

Comment:
Will this be the same for every pool? It doesn't seem like it belongs here, can just be in the code.

Reply:
no, pools are deprecated for different reasons, that's why we have to provide different description